cli/lib/commands/timeoutJobs.ts

```diff
@@ -39,7 +39,7 @@ export async function timeoutJobs({
       graph ?job {
         ?job a ${cc.Job} .
         ?job ${schema.actionStatus} ?status .
-        ?job ${dcterms.modified} ?modified .
+        ?job ${dcterms.modified}|${dcterms.created} ?modified .
```

Comment:
The fix is simple, to inspect either `modified` OR `created` and cancel those jobs